### PR TITLE
Simplify JBehave parameter converters

### PR DESCRIPTION
**Pull request by valfirst**

### CODE REVIEW

File: vividus-plugin-mongodb/src/main/java/org/vividus/bdd/converter/StringToBsonConverter.java [hunk 1]
```diff
@@ -16,22 +16,18 @@
...
-public class StringToBsonConverter extends AbstractParameterConverter<Bson>
+public class StringToBsonConverter extends FunctionalParameterConverter<Bson>
 {
-    @Override
-    public Bson convertValue(String value, Type type)
+    public StringToBsonConverter()
     {
-        String json = StringUtils.isNoneBlank(value) ? value : "{}";
-        return Document.parse(json);
+        super(value -> Document.parse(StringUtils.isNoneBlank(value) ? value : "{}"));
```
Comment (ikalinin1) [inline vividus-plugin-mongodb/src/main/java/org/vividus/bdd/converter/StringToBsonConverter.java:31]: It is out of scope, but could you please use StringUtils. defaultIfBlank instead explicit logic?

**Review by ikalinin1 — APPROVED** [review]

File: vividus-plugin-mongodb/src/main/java/org/vividus/bdd/converter/StringToBsonConverter.java (hunk 1 above)
Comment (ikalinin1) [reply vividus-plugin-mongodb/src/main/java/org/vividus/bdd/converter/StringToBsonConverter.java:31]: Oh, its noneBlank, please ignore

Comment (valfirst) [reply vividus-plugin-mongodb/src/main/java/org/vividus/bdd/converter/StringToBsonConverter.java]: I believe `isNoneBlank ` was used mistakenly

Comment (ikalinin1) [reply vividus-plugin-mongodb/src/main/java/org/vividus/bdd/converter/StringToBsonConverter.java]: Just checked, works for whitespaces too, so it's a valid comment :)

**Review by ikalinin1 — APPROVED** [review]

**Pull request merged**